app/models/grda_warehouse/tasks/push_clients_to_cas.rb

```diff
@@ -341,6 +345,16 @@ def value_display_for(key, value)
           rrh_successful_exit: 'RRH successful exit:',
           hmis_days_homeless_last_three_years: _('Days homeless in the last three years, from HMIS'),
           hmis_days_homeless_all_time: _('Total days homeless, from HMIS'),
+          am_ind_ak_native: "Race: #{::HUD.races['AmIndAKNative']}",
+          asian: "Race: #{::HUD.races['Asian']}",
+          black_af_american: "Race: #{::HUD.races['BlackAfAmerican']}",
+          native_hi_pacific: "Race: #{::HUD.races['NativeHIPacific']}",
+          white: "Race: #{::HUD.races['White']}",
+          female: "Gender: #{::HUD.genders[0]}",
+          male: "Gender: #{::HUD.genders[1]}",
+          no_single_gender: "Gender: #{::HUD.genders[4]}",
+          transgender: "Gender: #{::HUD.genders[5]}",
+          questioning: "Gender: #{::HUD.genders[6]}",
```

Comment:
These look like this on the CAS readiness page. It would require some changes to show them as one row, but I can do that if preferred.

<img width="640" alt="Screen Shot 2022-10-05 at 2 37 08 PM" src="https://user-images.githubusercontent.com/5333982/194136635-3f4608ae-833c-4375-b3e7-cd44c9abc066.png">